main.go

```diff
@@ -13,6 +15,6 @@ func main() {
 	log.SetFormatter(&log.TextFormatter{ForceColors: true})
 	log.SetOutput(colorable.NewColorableStdout())
 	if err := cmd.NewRootCmd().Execute(); err != nil {
-		log.Fatalln(err)
+		os.Exit(1)
```

Comment:
Does this mean we don't print the error messages anymore?

Reply:
No, cobra was printing them for us in the default implementation of `Execute()`, and then we were printing them again:

```console
$ ./bin/aks-engine get-versions -o yaml
Error: output format "yaml" is not supported
Usage:
  aks-engine get-versions [flags]

Flags:
  -h, --help             help for get-versions
  -o, --output string    Output format. Allowed values: human, json (default "human")
      --version string   Kubernetes version (optional)
      --windows          Kubernetes cluster with Windows nodes (optional)

Global Flags:
      --debug   enable verbose debug logs

FATA[0000] output format "yaml" is not supported
```

This change stops printing that last line.